shared/src/commonMain/kotlin/org/hyperskill/app/projects/domain/analytic/ProjectsListClickedProjectHyperskillAnalyticsEvent.kt

```diff
@@ -0,0 +1,52 @@
+package org.hyperskill.app.projects.domain.analytic
+
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticAction
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticEvent
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticPart
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticRoute
+import org.hyperskill.app.analytic.domain.model.hyperskill.HyperskillAnalyticTarget
+import ru.nobird.app.core.model.mapOfNotNull
+
+/**
+ * Represents an analytic event for clicking on a project in the projects-list.
+ *
+ * JSON payload:
+ * ```
+ * {
+ *     "route": "/track/1/projects",
+ *     "action": "click",
+ *     "part": "projects_list",
+ *     "target": "project",
+ *     "context":
+ *     {
+ *         "id": 1
+ *     }
+ * }
+ * ```
+ *
+ * @param trackId id of the track
+ * @param projectId is of the project
+ *
+ * @see HyperskillAnalyticEvent
+ */
+class ProjectsListClickedProjectHyperskillAnalyticsEvent(
```

Comment:
We also need to rename analytic events according to the new feature name `ProjectSelectionListFeature`

Reply:
and all other events